lib/src/json_serializable_generator.dart

```diff
@@ -58,6 +58,17 @@ class JsonSerializableGenerator
     // TODO: support overriding the field set with an annotation option
     var fieldsList = classElement.fields.where((e) => !e.isStatic).toList();
 
+    var undefinedFields =
+        fieldsList.where((fe) => fe.type.isUndefined).toList();
```

Comment:
I hate reiterating iterables – even trivially. Can have unintended perf and behavior affects.

So I just never do it.